### ISSUE TITLE
IE11, removeformatPasted, selected text is not replaced

### ISSUE THREAD

**Issue by dwittner** [Alex-D/Trumbowyg#497]
### Informations

**Browser + version**: IE11
**OS**: Windows 7

### How to reproduce the bug?
* use config option ```removeformatPasted: true```
* load page in IE(11)
* select some text in the editor
* paste content to the editor

__Result:__ The selected text is not replaced by the pasted content. Instead, the pasted content is inserted before the selected text.

### The following fix works for me:
```javascript
...
.on('paste', function (e) {
  if (t.o.removeformatPasted) {
    e.preventDefault();

    // Fix
    if (window.getSelection && window.getSelection().deleteFromDocument) {
      window.getSelection().deleteFromDocument();
    }

    try {
      // IE
      var text = window.clipboardData.getData('Text');
      ...
```

**Comment by Alex-D** [comment]
Hey!

Thanks for your bug report, can you make a PR of your fix? :)

**Comment by Alex-D** [comment]
Fix added in 2.8.0 which will be published soon.
